pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -1001,6 +1002,12 @@ def create_model(df_sets, df_parameters, default={}):
         doc="Weekly processing (e.g. clarification) capacity per storage site [bbl/week]",
         mutable=True,
     )
+    model.p_sigma_NetworkNode = Param(
+        model.s_N,
+        default=float("inf"),
```

Comment:
I suggest not using `float("inf")` for nodes that don't have a capacity constraint. The reason for this is that the model will be cluttered with equations that are redundant. For example, if node 1 doesn't have a capacity constraint, (and assuming our typical 52 weeks for the planning horizon), then the compiled model will have extra 52 constraints. My suggestion is to create a parameter, equivalent to NNA, PNA, etc, that tells PARETO what constraints should be built. We can do that either explicitly, that is another tab in the Spreadsheet, or we can build the set automatically based on the tab `NodeCapacities` and apply that set in `def NetworkNodeCapacityRule(model, n, t)`. I personally like building the parameter automatically, but that's up for discussion @melody-shellman @drouvenm your feedback is appreciated. 
Also we need to add a config option for "Node Capacities" and default it to `False`. This option should be used to check if we need to build parameter **`model.p_sigma_NetworkNode`** and constraint **`model.NetworkCapacity`**

Reply:
That's a fair point, I agree it is better to avoid creating constraints that are redundant. One thought: we could have two columns in NodeCapacity (a binary if the node has a capacity constraint and capacity value). These can generate the parameters and the constraints are only created for the nodes with a "1" binary. 

I know a binary indicator may not be intuitive for all our users, but it would eliminate having an extra tab in our already ~67 tab document.

Reply:
I like building the parameter automatically too. I think it's clear that if there's a value, it should be applied, and not if there's no value. I'll also set up a config option!

Reply:
I've added a config option for including the capacity at all. The current setup will skip the constraint is no value is entered in the input sheet. I've added a not about this in the excel file too.